error.go

```diff
@@ -221,3 +221,29 @@ var tooManyResultsError = &microerror.Error{
 func IsTooManyResults(err error) bool {
 	return microerror.Cause(err) == tooManyResultsError
 }
+
+var (
+	yamlConversionFailedErrorText = "error converting YAML to JSON:"
+)
+
+var yamlConvertingFailedError = &microerror.Error{
```

Comment:
```suggestion
var yamlConversionFailedError = &microerror.Error{
```

I guess it wasn't clear but you only changed the line I highlighted. The rest of the PR needs to be updated too.